aptos-move/framework/supra-framework/sources/coin.move

```diff
@@ -120,12 +120,16 @@ module supra_framework::coin {
     }
 
     /// Event emitted when some amount of a coin is deposited into an account.
-    struct DepositEvent has drop, store {
+    struct CoinDeposit has drop, store {
+		coin_type: String,
+		account: address,
```

Comment:
The formatting here is a bit off.

Reply:
@isaacdoidge , not sure what you mean by off. On my IDE it is perfectly aligned with the rest of the code.

Reply:
@isaacdoidge yes, all unit tests, cargo e2e move tests are passing. If there were relevant references, tests would have broken.

Reply:
Dr @sjoshisupra, your IDE might be converting tabs to spaces incorrectly. If you turn on the whitespace characters then you might see what I mean. GitHub renders the actual document, and you can see here that the fields are not aligned correctly.

Reply:
You can see here that you've used tabs while the original code uses spaces. Some IDEs renders tabs as 4 spaces but others render them as 8. It's simplest just to set your IDE to insert 4 spaces instead of tabs (VSCode does this by default). The first screenshot shows the render in VS Code (which renders tabs as 4 spaces) second screenshot shows the view in vim (renders tabs as 8 spaces), which renders tabs in the same way as GitHub.

<img width="590" alt="image" src="https://github.com/user-attachments/assets/6d9a6edb-cc91-4ca0-913a-1acdd1d263fa">
![image](https://github.com/user-attachments/assets/2286c556-ffbe-496b-803f-00222d5409e2)